safekeeper/src/send_wal.rs

```diff
@@ -257,9 +267,41 @@ impl WalSendersShared {
                     }
                     agg.ts = min(agg.ts, hs_feedback.ts);
                 }
+                let reply = standby_feedback.reply;
+                if reply.write_lsn != Lsn::INVALID {
+                    if reply_agg.write_lsn != Lsn::INVALID {
+                        reply_agg.write_lsn = Lsn::min(reply_agg.write_lsn, reply.write_lsn);
+                    } else {
+                        reply_agg.write_lsn = reply.write_lsn;
+                    }
+                }
+                if reply.flush_lsn != Lsn::INVALID {
+                    if reply_agg.flush_lsn != Lsn::INVALID {
```

Comment:
Since INVALID is defined as zero, wouldn't it be equivalent to just always do an Lsn::min here, rather than checking for INVALID?

Reply:
Unfortunately it doesn't work as 0 is is minimum u64 number, flush_lsb will never be assigned larger value without this check. Alternative is to use `Lsn::MAX` as default - in this case we can avoid this check. But it may cause problems in other places which expect `Lsn::INVALID` and not `Lsn::MAX`.

So I reprover the same logic as already used for PS feedback.